torch_frame/nn/encoder/stype_encoder.py

```diff
@@ -91,9 +91,9 @@ def init_modules(self):
                 raise ValueError(
                     f"{self.na_strategy} cannot be used on timestamp"
                     " columns.")
-            elif self.stype == stype.text_embedded:
+            elif self.stype == stype.embedding:
                 raise ValueError(f"Only the default `na_strategy` (None) "
-                                 f"can be used on embedded text columns, but "
+                                 f"can be used on {stype} columns, but "
```

Comment:
`self.stype` or just spell out embedding columns.

`stype` is not defined in this scope.